Tests/StreamChatUITests/SnapshotVariant.swift

```diff
@@ -5,56 +5,89 @@
 @testable import StreamChatUI
 import XCTest
 
-enum SnapshotVariant: String, Hashable, CaseIterable {
-    case small
-    case large
-    case extraExtraExtraLarge
-    
-    static var userInterfaceStyleCases: [SnapshotVariant] {
-        [small, large]
+/// A Snapshot Variant is a combination of SnapshotTraits, that will result in a snapshot test with multiple UITraitCollection's.
+struct SnapshotVariant {
+    let snapshotTraits: [SnapshotTrait]
+    var snapshotName: String {
+        snapshotTraits.map(\.name).joined(separator: ".")
     }
     
     var traits: UITraitCollection {
-        var traits = [
-            UITraitCollection(displayScale: 1),
-            contentSizeCategoryTrait
-        ]
-        
+        UITraitCollection(traitsFrom: [UITraitCollection(displayScale: 1)] + snapshotTraits.map(\.trait))
+    }
+}
+
+/// A Snapshot Trait is usually just a combination of a UITraitCollection and it's name.
+struct SnapshotTrait {
+    let name: String
+    let trait: UITraitCollection
+}
+
+extension SnapshotVariant {
+    // MARK: - Combinations
+    
+    static let all: [SnapshotVariant] = {
+        [smallAndDark, largeAndLight, extraExtraExtraLargeAndLight]
+    }()
+    
+    static let onlyUserInterfaceStyles: [SnapshotVariant] = {
+        [smallAndDark, largeAndLight]
```

Comment:
This is not correct. This should be `largeAndLight` and `largeAndDark` -> only user inteface style should change, not the size

Reply:
Changes done ✅